components/link/link.js

```diff
@@ -1,7 +1,28 @@
 import '../colors/colors.js';
 import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { classMap} from 'lit-html/directives/class-map.js';
 import { ifDefined } from 'lit-html/directives/if-defined.js';
-import { linkStyles } from './link-styles.js';
```

Comment:
Note: I collapsed the styles into here and `export`ed them while I was at it. This aligns with our best practice for not having unnecessary imports. I did a code search for `linkStyles` that didn't turn up anything, so think this is safe.